keyboards/keychron/q2/chconf.h

```diff
@@ -16,7 +16,7 @@
 
 /*
  * This file was auto-generated by:
- *    `qmk chibios-confmigrate -i keyboards/acheron/arctic/chconf.h -r platforms/chibios/common/configs/chconf.h`
+ *    `qmk chibios-confmigrate -i keyboards/keychron/q2/chconf.h -r platforms/chibios/boards/common/configs/chconf.h`
```

Comment:
Just delete this whole comment block. It's injected when it's been migrated, but serves no purpose when it's a new board. No need to keep it.

Reply:
Thanks a lot. I got it.